shifthelper/checks.py

```diff
@@ -56,7 +56,15 @@ def __init__(self, category, *args, **kwargs):
 
     def check(self):
```

Comment:
I think this function should not be defined inside here everytime the check runs, right?